### ISSUE TITLE
Renaming `Typography*` to `Font*`

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/ouds-ios#322]
Fonts tokens are the latest raw + semantic tokens to sync with Figma.

There's a little preparation work to do before on your side to receive files generated from OUDS Tokenator.

- `TypographyRawTokens+Values.swift` must be renamed to `FontRawTokens+Values.swift` (and so the extension name as well)
- `OUDSTheme+TypographySemanticTokens.swift` must be renamed to `OUDSTheme+FontSemanticTokens.swift` (and so the extension name as well)
  - The types in this file must be renamed too: `Typography*` → `Font` (e.g. `TypographyFontFamilySemanticToken` → `FontFamilySemanticToken`)
- Basically, most of the things that are named `Typography*` should be `Font*`; "typography" doesn't exist on the tokens side.
